Proofreading MO Guide (#11605)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_CRNN_From_Tensorflow.md

# Converting a TensorFlow CRNN Model {#openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_CRNN_From_Tensorflow}

This tutorial explains how to convert a CRNN model to Intermediate Representation (IR).

> You will find on GitHub several public versions of TensorFlow CRNN model implementation. This tutorial explains how to convert the model from

Author: ```suggestion
There are several public versions of TensorFlow CRNN model implementation available on GitHub. This tutorial explains how to convert the model from
```

the [https://github.com/MaybeShewill-CV/CRNN_Tensorflow](https://github.com/MaybeShewill-CV/CRNN_Tensorflow) repository to IR. 

> If you have another implementation of CRNN model, it can be converted to the IR in similar way. You need to get inference graph and run Model Optimizer on it.

Reviewer: ```suggestion
If you have another implementation of CRNN model, it can be converted to OpenVINO IR in a similar way. You need to get inference graph and run Model Optimizer on it.
```
